backend/development.ini.sample

```diff
@@ -420,6 +422,15 @@ search.engine = simple
 # content for only file with those mimetype, if set to empty, allow any mimetype, default value
 # allow most text-based content type.
 ; search.elasticsearch.ingest.mimetype_whitelist = text/markdown,text/plain
+# if you set ingest mimetype blacklist to list of separated mimetype by "," , you will not index file
+# content with those mimetype (even in included in whitelist), if set to empty, do nothing, default
+# value to "" meaning no blacklist.
+; search.elasticsearch.ingest.mimetype_blacklist = text/markdown,application/gzip,application/zip
```

Comment:
could you remove text/markdown from the example? It's not an expected use case to activate plain-text search and to exclude markdown documents.